build.cake

```diff
@@ -405,16 +406,37 @@ Task("Restore")
     RunTool(env.DotNetCommand, "restore OmniSharp.sln", env.WorkingDirectory)
         .ExceptionOnError("Failed to restore projects in OmniSharp.sln.");
 
+    // Restore test assets
```

Comment:
I plan to refactor at somepoint... newer cake versions make some of this stuff extra un-needed.